src/Sylius/Component/Core/spec/Model/TaxonomySpec.php

```diff
@@ -25,7 +31,7 @@ function it_is_Sylius_Taxonomy()
         $this->shouldImplement('Sylius\Component\Taxonomy\Model\TaxonomyInterface');
     }
 
-    function it_should_not_path_defined_by_default()
+    function it_should_not_have_path_defined_by_default()
     {
         $this->getRoot()->getPath()->shouldReturn(null);
     }
```

Comment:
I think I need some support in understanding why now is this spec failing ;<
@pjedrzejewski @michalmarcinkowski ? :D